src/org/zaproxy/zap/extension/pscanrulesAlpha/resources/help/contents/pscanalpha.html

```diff
@@ -9,12 +9,6 @@
 <H1>Passive Scan Rules - Alpha</H1>
 The following alpha quality passive scan rules are included in this add-on:
 
-<H2>ASP.NET ViewState Disclosure</H2>
```

Comment:
The scanner was not promoted yet, these are raised by Base64 Disclosure.

Reply:
Thanks, I totally missed that!

Reply:
Does it worth put the name of the scanner after the alert name? (or add a note somewhere?)

Reply:
I was wondering that yesterday, while reading through these.

Something like `ASP.NET ViewState Disclosure (Base64Disclosure)`

Reply:
Looks good. I think we just need to do that in the help(?), in the other places we include the ID of the scanner or its name (e.g. Alerts tab with Source field).

Reply:
Yup. I’ll put together a PR for Alpha shortly and we can discuss further in review of that.

Reply:
Great, thanks!